socrates/lib/activities/index.js

```diff
@@ -101,8 +101,8 @@ app.post('/submit', (req, res, next) => {
 
 // for management tables:
 
-app.get('/fromWaitinglistToParticipant/:roomType/:nickname', (req, res) => {
-  const duration = 2;
+app.get('/fromWaitinglistToParticipant/:roomType/:nickname/:duration', (req, res) => {
```

Comment:
Why would that come through the URL? It's already in the waitinglist event and needs to be carried over from there. No point in sending it from the management page, imho.